index.html

```diff
@@ -156,12 +156,12 @@ <h1 id="manifesto" class="title text-center">Our <span>Manifesto</span></h1>
 -->
 	      <p class="lead">Web apps today have incredible power</p>
 	      <p>
-	      When we use most modern web applications,
-	      the majority of computation and storage over your data happens
-	      on servers owned and operated by the developer of the website and their partners.
+	      When we use modern web applications,
+	      the majority of computation and storage of our data happens
+	      on servers owned and operated by big companies.
 	      For consumers, the cloud is synonymous with a giant computer you don't have any control over.
-	      You are trusting that the website, the associated company, 
-	      its potentially many employees, and their partners protect your interests.
+	      You are trusting that the website, the data center, 

```

Comment:
We'll still want to use data centers in the common case. But the terms of service of a data center are different than that for a web app